tracker/tracker-assist/src/Assist.ts

```diff
@@ -247,11 +258,15 @@ export default class Assist {
       this.agents[id]?.onDisconnect?.()
       delete this.agents[id]
 
+      if (Object.keys(this.agents).length === 0 && recordingState.isActive) {
+        recordingState.denyRecording()
```

Comment:
What if there are still agents, but the one recording one disconnected?
It should be rather `denyRecording(id)`, no?
BTW, I think it is better to split  `stopAgentRecording(id)` and  `denyRecording` since those are different actions.